### PR TITLE
[FIX] document_knowledge: Error typo res_config

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

File: setup/_metapackage/setup.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
     version=version,
     install_requires=[
         'odoo-addon-document_knowledge>=16.0dev,<16.1dev',
+        'odoo-addon-document_page>=16.0dev,<16.1dev',
```
Comment (pedrobaeza) [inline setup/_metapackage/setup.py]: This shouldn't go here

**Review by AntoniRomera — APPROVED** [review]

**Review by MRomeera — APPROVED** [review]
ok

**Review by max3903 — APPROVED** [review]

File: setup/_metapackage/VERSION.txt [hunk 2]
```diff
@@ -1 +1 @@
-16.0.20221111.0
\ No newline at end of file
+16.0.20221207.0
```
Comment (pedrobaeza) [inline setup/_metapackage/VERSION.txt]: And why this change?

Comment (FernandoRomera) [reply setup/_metapackage/VERSION.txt]: @pedrobaeza 
I don't know, I guess some process must create the setup directory.
How do you think it would be better to act: recover the previous value, or launch some command to correct it.

Comment (pedrobaeza) [reply setup/_metapackage/VERSION.txt]: Rebase and remove this change

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
It includes a merge commit in the branch.

File: setup/_metapackage/VERSION.txt (hunk 2 above)
Comment (pedrobaeza) [reply setup/_metapackage/VERSION.txt]: This is still happening.
